### PR TITLE
port cycle 1 locations that kneel to the cost framework

### PR DESCRIPTION
**Pull request by zacchiro**
Note: the implementation of Street of the Sisters has reduced a lot as part of
this pass, as it was doing lot of manual stuff for functionalities that were
available from the core game engine. Its code might deserve additional review
scrutiny.

### CODE REVIEW

File: server/game/cards/locations/02/streetofthesisters.js [hunk 1]
```diff
@@ -1,48 +1,25 @@
...
-                    { text: 'Kneel faction card', method: 'kneel' },
-                    { text: 'Cancel', method: 'cancel' }
-                ]
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                afterChallenge: (e, challenge) =>
+                    challenge.winner === this.controller
+                    && challenge.challengeType === 'power'
+                    && challenge.strengthDifference < 5
```
Comment (cryogen) [inline server/game/cards/locations/02/streetofthesisters.js]: The logic of this has been reversed

**Comment by zacchiro** [comment]
On Sun, Feb 26, 2017 at 04:53:52AM -0800, Stuart Walsh wrote:
> +            when: {
> +                afterChallenge: (e, challenge) =>
> +                    challenge.winner === this.controller
> +                    && challenge.challengeType === 'power'
> +                    && challenge.strengthDifference < 5
> 
> The logic of this has been reversed

Nice catch!

When are we gonna get scenario-based testing, again? :-)

Fixed and pushed an updated PR.

Cheers.

**Comment by cryogen** [comment]
I merged your other PR which now conflicts with this, sorry :)

(also merged the integration test stuff :P)

**Comment by zacchiro** [comment]
On Sun, Feb 26, 2017 at 05:10:29AM -0800, Stuart Walsh wrote:
> I merged your other PR which now conflicts with this, sorry :)

Heh. That will teach me to parallelize too much :-P

Fixed.

> (also merged the integration test stuff :P)

\o/

**Pull request merged**